openvpn/tunnel/setup_tun_linux.go

```diff
@@ -113,6 +121,24 @@ func (service *LinuxTunDeviceManager) deviceExists(device tunDevice) (exists boo
 	return false, err
 }
 
+func (service *LinuxTunDeviceManager) deviceUsed(device tunDevice) (exists bool, err error) {
+	contents, err := ioutil.ReadFile("/sys/class/net/" + device.Name + "/carrier")
+	if err != nil {
+		return false, err
+	}
+
+	value, err := strconv.Atoi(string(contents))
```

Comment:
Maybe we can compare it with a string to avoid extra conversion: 
```suggestion
	return string(contents) == "1", nil
```

Reply:
its sorta defensive, in case contents is not a number..